### ISSUE TITLE
Unable to build with github.com/shirou/gopsutil/v3 v3.22.10

### ISSUE THREAD

**Issue by gibmat** [minio/madmin-go#157]
I am working on packaging this library for use in Debian, and have encountered the following build error when using github.com/shirou/gopsutil/v3 v3.22.10 (the current version of that library and recently updated in Debian):

```
# github.com/minio/madmin-go
src/github.com/minio/madmin-go/health.go:328:36: undefined: host.Warnings
```

It appears that commit https://github.com/shirou/gopsutil/commit/dbc0f20fe38841a0c1509de9dba5084b7c2c09d9 moved `host/types.go` -> `internal/common/warnings.go`.

**Comment by harshavardhana** [comment]
why can't you use? 

>         github.com/shirou/gopsutil/v3 v3.22.9

as per go.mode?

**Comment by gibmat** [comment]
Debian policy requires that all packages can build without any Internet access. This means that all golang libraries are firstly packaged, and then when needed to build another package they are added to the GOPATH automatically. In Debian, the github.com/shirou/gopsutil package has already been updated to 3.22.10, and you can't have different versions of the same package in unstable/stable/etc.

It looks like gopsutil has partially reverted that change in https://github.com/shirou/gopsutil/pull/1379, so this may resolve itself in the next release of that library.

**Comment by harshavardhana** [comment]
This has been fixed via [cbae3a3](https://github.com/mrusme/madmin-go/commit/cbae3a351b98eecd25f83c07ab14dc4295caddbe)
